### PR TITLE
Removing files section from nuspec so we generate a full release

### PR DESCRIPTION
**Pull request by secondsun**
I removed the files section from nuspec.  This was causing the library to generate an empty package and this meant nothing was getting deployed.

### CODE REVIEW

**Comment by secondsun** [comment]
@edewit wdyt?

File: Dist/FHSDK.nuspec [hunk 1]
```diff
@@ -51,7 +51,8 @@
 
        <group targetFramework="monoandroid">
         <dependency id="Newtonsoft.Json" version="8.0.3" />
-        <dependency id="Unity" version="4.0.1" />
```
Comment (edewit) [inline Dist/FHSDK.nuspec]: we need Unity

**Comment by edewit** [comment]
Do you know why this is needed suddenly?

**Comment by edewit** [comment]
added in a432dc95a2839b3d4b7d7087de4215bd4e3bf030

**Pull request closed without merging**